lib/core/src/lib/dialogs/unsaved-changes-dialog/unsaved-changes-dialog.component.spec.ts

```diff
@@ -16,39 +16,66 @@
  */
 
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { CoreTestingModule, UnsavedChangesDialogComponent } from '@alfresco/adf-core';
+import { CoreTestingModule, StorageService, UnsavedChangesDialogComponent } from '@alfresco/adf-core';
 import { By } from '@angular/platform-browser';
 import { DebugElement } from '@angular/core';
-import { MatDialogClose } from '@angular/material/dialog';
+import { MAT_DIALOG_DATA, MatDialogClose } from '@angular/material/dialog';
+import { UnsavedChangesDialogData } from './unsaved-changes-dialog.model';
 
-describe('UnsavedChangesDialog', () => {
+describe('UnsavedAiChangesDialog', () => {
```

Comment:
Should this really be 'UnsavedAiChangesDialog'? Usually we use the same name as the component, which is just UnsavedChangesDialog